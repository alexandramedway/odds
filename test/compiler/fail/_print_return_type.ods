do print("Hello, world!") + 42
