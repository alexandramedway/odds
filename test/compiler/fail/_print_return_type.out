[31mSemantic error[0m, line 1:
  Invalid use of binary operator '+' with types String and Num
